tests/expand-manifest.jsonld

```diff
@@ -1225,6 +1227,22 @@
       "purpose": "Verifies that an exception is raised on expansion when processing an invalid context attempting to define @container on a keyword",
       "input": "expand/e042-in.jsonld",
       "expect": "keyword redefinition"
+    }, {
+      "@id": "#te043",
+      "@type": ["jld:NegativeEvaluationTest", "jld:ExpandTest"],
+      "name": "Term definition with @id: @type",
+      "purpose": "Expanding term mapping to @type uses @type syntax now illegal",
+      "input": "expand/e043-in.jsonld",
```

Comment:
The label `#te043` here doesn't match the input file name prefix `e043` (same for `e044`) -- which seems to break the naming convention, so I suspect this is a typo. Just checking here, was this intentional or a typo?

Reply:
Tests all of an identifier such as `#txxxx` with corresponding files `expand/xxxx-in.jsonld` and `expand/xxx-out.jsonld`. In this case the `#te043` is consistent with the file named `e043-in` and follows the same pattern as that for `#te042`.

Reply:
I think the convention has been the test ids have a `t` prefix, the file names don't.  Is that what you meant?

Reply:
Yes, the ids all start with a "t", the filename's don't. This convention was established in the 1.0 days. I established the practice if replacing one or two leading zeros with letters to help separate out the new tests by broad feature categorization, which helps in filtering as well.

The `#texxx` tests typically are for errors, some of which previously came from a separate error manifest.